libs/data-mapper/src/lib/utils/ReactFlow.Util.ts

```diff
@@ -178,6 +178,26 @@ const convertSourceToReactFlowParentAndChildNodes = (
     return reactFlowNodes;
   }
 
+  const sourceNodesCopy = [...combinedSourceSchemaNodes];
+  sourceNodesCopy.filter((node) => node.nodeProperties.includes(SchemaNodeProperty.Repeating));
+  const sourceKeySet: Set<string> = new Set();
+  sourceNodesCopy.forEach((node) => sourceKeySet.add(node.key));
+  const widthDict: Map<string, number> = new Map<string, number>();
+  let maxSize = 200;
```

Comment:
Nit: Maybe add some of these numbers to constants and then reference them here?

Reply:
done!